### PR TITLE
Fix CI errors

### PR DESCRIPTION
**Pull request by mpetrun5**
## Description

Fix lint and test errors 

## Related Issue Or Context

Closes: #<issue>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -40,16 +33,18 @@ func TestValidateConfig(t *testing.T) {
...
+		Endpoint: "endpoint",
+		From:     "",
+	}
+
 	err := valid.Validate()
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	err = missingType.Validate()
```
Comment (P1sar) [inline config/config_test.go:48]: ye i guess there were a bug in test

**Review by P1sar — APPROVED** [review]

File: e2e/evm/test.go [hunk 2]
```diff
@@ -108,7 +108,7 @@ func (s *IntegrationTestSuite) TestDeposit() {
 	}
 	//wait for vote log event
 	proposalEvent := "ProposalEvent(uint8,uint64,uint8,bytes32,bytes32)"
-	evts, err := s.client2.FetchEventLogs(context.Background(), s.bridgeAddr, proposalEvent, b, ba)
+	evts, _ := s.client2.FetchEventLogs(context.Background(), s.bridgeAddr, proposalEvent, b, ba)
```
Comment (waymobetta) [inline e2e/evm/test.go:111]: why is error being ignored here?

I see that within the `FilterLogs` the error could occur and would get returned back to the calling code, `FetchEventLogs`, then returned back here but the error is being ignored

Comment (waymobetta) [reply e2e/evm/test.go:111]: I guess it may not matter as much because this is just a test

Comment (mpetrun5) [reply e2e/evm/test.go:111]: I answered a minute to late :smile:

**Pull request merged**